apitools/org.eclipse.pde.api.tools.tests/src/org/eclipse/pde/api/tools/anttasks/tests/ApiToolingApiuseAntTaskTests.java

```diff
@@ -126,8 +126,6 @@ public void test3() throws Exception {
 
 	/**
 	 * Tests that a use scan will find illegal use problems that can be filtered
-	 *
```

Comment:
Does the removal of such empty lines survive a subsequent formatting with the current settings?
If not, please revert it.

Reply:
Formatting does not add empty lines.  tested.